commercetools-models/src/main/java/io/sphere/sdk/apiclient/expansion/ApiClientExpansionModel.java

```diff
@@ -0,0 +1,11 @@
+package io.sphere.sdk.apiclient.expansion;
+
+import io.sphere.sdk.apiclient.ApiClient;
+
+public interface ApiClientExpansionModel<T> {
```

Comment:
What's the purpose of this interface? It shouldn#t be necessary since you can't expand an `ApiClient`!?

Reply:
this interface is used some generated classes such as `ApiClientByIdGet`, if removed i ll have to write these classes.